app/models/user.rb

```diff
@@ -1,10 +1,13 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
* Good job so far but you are yet to implement two requirements 

1. Your application Ask for confirmation of email
2. A user can reset the password in the application. when i tried resetting the password i got an error. below is a screenshot ⬇️ 
![image](https://user-images.githubusercontent.com/52497246/186871781-315bf78c-5dde-4f4b-bbf4-68d646f4a606.png)